packages/trace-viewer/src/ui/workbenchLoader.tsx

```diff
@@ -213,9 +213,10 @@ export const WorkbenchLoader: React.FunctionComponent<{
       <button onClick={() => {
         const input = document.createElement('input');
         input.type = 'file';
+        input.accept = 'application/zip';
```

Comment:
Does that work on Windows with another type that you mentioned or are you breaking it?